tests/unit/test_cli_utils.py

```diff
@@ -70,7 +70,7 @@ def covered_func():
 
 
 def test_cover_packit_exception_decorator_exit_code_other_exception():
-    flexmock(sys).should_receive("exit").with_args(3)
+    flexmock(sys).should_receive("exit").with_args(4)
```

Comment:
Could you explain to me why this has changed?

Reply:
https://github.com/packit-service/packit/pull/364/commits/692e387d88ee0f3b97b32cd902b480ac87e38c41

Reply:
this got broken over here: https://github.com/packit-service/packit/pull/342
I fixed it in https://github.com/packit-service/packit/pull/364/commits/692e387d88ee0f3b97b32cd902b480ac87e38c41

hence, needs rebase, sorry